compliance/virtualmachines/loadgen/main.go

```diff
@@ -0,0 +1,95 @@
+package main
```

Comment:
It feels a bit strange that a new binary is located within compliance/virtualmachines. I think, this should be clearly decoupled from the rest of production code.